.github/workflows/actions.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           - t12
           - t16
           - t18
-          - t8
+          - t8;commando8
```

Comment:
It should be sufficient to the target to the "build-only" actions (bellow), as it does not change anything fundamentally.